### PR TITLE
Added popup task

### PR DESCRIPTION
**Pull request by mksen85**
### popup

[Demo](https://mksen85.github.io/popup) |
[Code base](https://github.com/mksen85/popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by mksen85** [comment]
Hello, I have self-checked my code, can you review it, please?

File: submissions/mksen85/index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                                                                                        class="image"
+                                                                                        src="./img/calendar.png"
+                                                                                        alt="calendar"
+                                                                                        tabindex="0"><span>Calendar</span></a>
+                                                                </li>
+                                                                <li>
+                                                                        <a href="" class="submenu_link"><img
+                                                                                        class="image"
+                                                                                        src="./img/mail.png"
+                                                                                        alt="mail"><span>Mail</span></a>
```
Comment (A-Ostrovnyy) [inline submissions/mksen85/index.html:44]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/mksen85/index.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                                                                                        alt="photo"><span>Photos</span></a>
+                                                                </li>
+                                                                <li>
+                                                                        <a href="" class="submenu_link"><img
+                                                                                        class="image"
+                                                                                        src="./img/apps.png"
+                                                                                        alt="apps"><span>Apps</span></a>
+                                                                </li>
+                                                                <li>
+                                                                        <a href="" class="submenu_link"><img
```
Comment (A-Ostrovnyy) [inline submissions/mksen85/index.html:101]:
```suggestion
     <a href="#" class="submenu_link"><img
```

File: submissions/mksen85/index.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                                                                                </div>
+
+                                                                        </ul>
+                                                                </div>
+                                                        </div>
+                                                </ul>
+                                        </li>
+                                        <li>
+                                                <a href="" class="menu_link"><img src="./img/bell.png" alt="bell"
+                                                                width="24px" height="24px"></a>
```
Comment (A-Ostrovnyy) [inline submissions/mksen85/index.html:168]: Add styles on focus state.

File: submissions/mksen85/style.css [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    border: 1em;
+  }
+
+  .menu_list li{
+    list-style: none;
+  }
+
+  menu li{
+    position: relative;
+  }
```
Comment (A-Ostrovnyy) [inline submissions/mksen85/style.css:64]: `menu` tag?)
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

File: submissions/mksen85/style.css [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+ .button_more{
+  position: relative;
+  display: grid;
+  width: 100%;
+  grid-column: 1 / 4;
+  color: rgb(99, 93, 93);
+  font-weight: bold; 
+  position: relative;
+  width: 100%;
+  }
```
Comment (A-Ostrovnyy) [inline submissions/mksen85/style.css:170]: Use prettier.io for it to format the code for you. Turn it on in your code editor or install a plugin.

**Pull request closed without merging**